### DISCUSSION TITLE
something about cdn

### DISCUSSION THREAD

**Discussion by zhuwowowo** [vuetifyjs/vuetify#12079] [General] [answered]
- Since I deploy web in an intranet environment, I cannot use cdn.
- I added vuetify with vue ui.
- And then i find https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css in google's network.
- This make some Components works incorrect.
- So how to fix it?

**Comment by jacekkarczmarczyk** [comment, marked as answer]
Install icons as a dependency:

https://vuetifyjs.com/en/customization/icons/#installing-iconfonts

**Reply by zhuwowowo** [reply]
thanks
